_fixtures/testinline.go

```diff
@@ -7,6 +7,7 @@ func inlineThis(a int) int {
 	return z + a
 }
 
+//go:noinline
```

Comment:
Can you undo this and the changes related to this? I have a different fix for this in 1.20 that I prefer, it doesn't matter if tests don't pass on 1.20.

Reply:
Sure, done.